### ISSUE TITLE
FizzBuzzEnterpriseEdition does not work when Java is not installed

### ISSUE THREAD

**Issue by ekolis** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#507]
I don't have Java installed because my cousin in law's roommate's sister said it was a virus. I only use .NET, Perl, and Erlang; all other programming languages are too dangerous. Please make FizzBuzzEnterpriseEdition run on at least one of these platforms, preferably all three in case two of them are compromised. Thank you! 🙂

**Comment by heleg** [comment]
> Please make FizzBuzzEnterpriseEdition run on at least one of these platforms

:+1: 
It would be great if platform could be selected during installation, or even better if it could be changed afterwards.

**Comment by ekolis** [comment]
Yeah, there could be a Platform.xml file which specifies which platform we're using. Or maybe a priority list of platforms, in case there's an error loading one.

**Comment by CTimmerman** [comment]
I vote for vertical integration; it works for Apple and Tesla. We should brainstorm over market research and A/B test potential brand names to see what resonates best with our clients.

**Comment by StefanLobbenmeier** [comment]
We should avoid splitting the codebase to support multiple platforms and use polyglot coding instead: https://rosettacode.org/wiki/Polyglot
